### PR TITLE
@eessex => Super Article redesign (nav, footer, responsive)

### PR DESCRIPTION
**Pull request by kanaabe**
Closes https://github.com/artsy/force/issues/336 https://github.com/artsy/force/issues/337

PR-ing for now, need to do a round of cleanup and then I'll let you know when it's ready for review!

Header desktop:
super-article-header

Header mobile:
super-article-header-mobile

Footer desktop:
image

Footer mobile:
image

### CODE REVIEW

**Comment by eessex** [comment]
! looks great

File: components/article/client/super_article.coffee [hunk 1]
```diff
@@ -1,45 +1,81 @@
...
+
+      @$body.removeClass 'is-open'
+      initCarousel @$('.article-sa-sticky-related-container'),
+        imagesLoaded: true
+        wrapAround: true
+        advanceBy: 1
+        cellAlign: 'left'
+      , (carousel) =>
+        # Wait for the callback to set carousel
+        @carousel = carousel
```
Comment (kanaabe) [inline components/article/client/super_article.coffee]: `initCarousel` returns a Promise when using the `imagesLoaded` option -- by passing in a third argument, the callback, you can wait for the images to load, and set `@carousel` in the view after.

File: components/article/stylesheets/fullscreen_article.styl [hunk 2]
```diff
@@ -111,36 +101,59 @@ body.body-fullscreen-article
...
-  width 750px
   margin-bottom 0px
-  @media screen and (min-width 1100px)
-    width 900px
-  @media screen and (min-width 1500px)
-    font-size 25px
 
-.article-sa-fullscreen-logos
-  width 750px
+.article-fullscreen
```
Comment (kanaabe) [inline components/article/stylesheets/fullscreen_article.styl]: Just trying to clean up this file by moving all thing media queries to the bottom.

**Comment by kanaabe** [comment]
Cleanup should be good to go now, let me know if any of this is unclear and we can go over it together.

File: components/article/client/super_article.coffee [hunk 3]
```diff
@@ -1,45 +1,81 @@
...
+  hideNav: ->
+    @$superArticleNavToc.css 'height', '0px'
+    @$superArticleNavToc.removeClass('visible')
+    @$body.removeClass 'is-open'
 
-  setupWaypoints: ->
-    $stickyHeader = @$('.article-sa-sticky-header')
-    return unless $stickyHeader.length
+  setStickyNav: =>
+    if window.matchMedia('(max-width: 900px)').matches
```
Comment (eessex) [inline components/article/client/super_article.coffee]: nice, haven't seen this method of screen querying!

**Pull request merged**